plugin/handler.js

```diff
@@ -64,6 +64,24 @@ var bowerHandler = function (compileStep, bowerTree, bowerHome) {
   var localCache = Bower.list(null, {offline: true, directory: bowerHome, cwd: cwd});
   var bowerDependencies = _.chain(getDependencies(localCache)).sortBy("depth").reverse().value();
```

Comment:
It depends. If the goal if this sort is just to avoid dependency order problems, it's not anymore. But if it's there to allow developers some kind of ordering strategy based on deepness declaration and is used as a feature, it should stay. But in this case, I'd suggest some cleaner way to achieve it with the "override" property in the bower.json file.

So I think it could be removed.